modules/ducktests/tests/ignitetest/services/utils/ignite_aware.py

```diff
@@ -156,10 +164,9 @@ def await_event(self, evt_message, timeout_sec, from_the_beginning=False, backof
         :param backoff_sec: Number of seconds to back off between each failure to meet the condition
                 before checking again.
         """
-        assert len(self.nodes) == 1
```

Comment:
Was there a reason to have the assert?

Reply:
@timoninmaxim It was implemented partially by me, assert was used to check it was used properly (applicable only for single-node services)